pkg/operator/starter.go

```diff
@@ -97,6 +98,27 @@ func RunOperator(ctx context.Context, controllerConfig *controllercmd.Controller
 		return err
 	}
 
+	desiredVersion := status.VersionForOperatorFromEnv()
+	missingVersion := "0.0.1-snapshot"
+
+	// By default, this will exit(0) the process if the featuregates ever change to a different set of values.
+	featureGateAccessor := featuregates.NewFeatureGateAccess(
+		desiredVersion, missingVersion,
+		configInformers.Config().V1().ClusterVersions(), configInformers.Config().V1().FeatureGates(),
+		controllerConfig.EventRecorder,
+	)
+	go featureGateAccessor.Run(ctx)
+	go configInformers.Start(ctx.Done())
```

Comment:
you don't need to start informers in a separate goroutine
`configInformers.Start(ctx.Done())` works

Reply:
`configInformers` are started again in L367 (https://github.com/openshift/cluster-openshift-apiserver-operator/blob/e8930013a372a80360467c103a8ddfc1512d3d5f/pkg/operator/starter.go#L367)

please fix it.

Reply:
The config informers must be started here so this control loop runs.  Doing what you suggest and avoiding a second restart will cause other callers of the shareinformerfactory that start informers against *other* resources to never be started.

Reply:
okay, ic, `configInformers` is a factory not an informer :)